yoga/BitUtils.h

```diff
@@ -11,7 +11,7 @@
 #include <cstdio>
 #include <cstdint>
 
-#include <yoga/YGEnums.h>
+#include "YGEnums.h"
```

Comment:
Meta's Buck build is set up so that public headers `{Yoga,YGEnums,YGMacros,YGValue}.h` must be accessed using `yoga/` [header namespace](https://buck.build/rule/cxx_library.html#exported_headers).

So we unfortunately cannot access them as relative paths, or via traversal.

It is a bit verbose, and maybe a bit breaking, but an option I had looked at was to fully move files like `yoga/Yoga.h` to `yoga/include/yoga/Yoga.h`.

Reply:
OK, I see that. Now I just add tow new files to support SwiftPM.